pcaplua.c

```diff
@@ -82,6 +104,41 @@ static int next (lua_State *L) {
 	return 3;
 }
 
+/** gets one packet
+ * @memberof pcap
+ * @return packet data, timestamp, offwire length
+ */
+static int next_ex (lua_State *L) {
+	l_pcap *p = check_pcap(L,1);
+	struct pcap_pkthdr *ph;
+	const u_char *d;
+	int res = pcap_next_ex (p->pcap, &ph, &d);
+	if (0 == res) {
+		// live capture packet buffer timeout
+		lua_pushboolean(L, 1);
+		lua_pushnil(L); // no packet
+		return 0;
```

Comment:
these two values are discarded by the `return 0`

Reply:
Ah, yes. Sorry.